crypto/evp/cmeth_lib.c

```diff
@@ -14,6 +14,10 @@
 #include "internal/provider.h"
 #include "evp_locl.h"
 
+#if OPENSSL_API_3
```

Comment:
If this define is around the whole source file - then its a pity it cant be disabled via build.info

Reply:
Oh wait...  I did a modification in `Configure` for exactly this purpose (I added `$config{minapi}`).  So yeah, fix coming up!